doc/developer-guides/hld/usb-virt-hld.rst

```diff
@@ -21,25 +21,27 @@ emulation of three components, described here and shown in
 
 - **xHCI DM** (Host Controller Interface) provides multiple
   instances of virtual xHCI controllers to share among multiple User
-  OSes, each USB port can be assigned and dedicated to a VM by user
+  VMs, each USB port can be assigned and dedicated to a VM by user
   settings.
 
 - **xDCI controller** (Device Controller Interface)
   can be passed through to the
-  specific User OS with I/O MMU assistance.
+  specific User VM with I/O MMU assistance.
 
 - **DRD DM** (Dual Role Device) emulates the PHY MUX control
   logic. The sysfs interface in a User VM is used to trap the switch operation
-  into DM, and the the sysfs interface in the Service VM is used to operate on the physical
-  registers to switch between DCI and HCI role.
-
-  On Intel Apollo Lake platform, the sysfs interface path is
-  ``/sys/class/usb_role/intel_xhci_usb_sw/role``. If user echos string
-  ``device`` to role node, the usb phy will be connected with xDCI controller as
-  device mode. Similarly, by echoing ``host``, the usb phy will be
-  connected with xHCI controller as host mode.
-
-An xHCI register access from a User VM will induce EPT trap from the User VM to
+  into DM, and the sysfs interface in the Service VM is used to operate on the
+  physical registers to switch between DCI and HCI role.
+
+  On Apollo Lake platforms, the sysfs interface path is
+  ``/sys/class/usb_role/intel_xhci_usb_sw/role``. If the user echos the string
+  ``device`` to the role node, the USB PHY will be connected with the xDCI
+  controller as
+  device mode. Similarly, by echoing ``host``, the USB PHY will be
+  connected with the xHCI controller as host mode.
+
+An xHCI register access from a User VM will induce an EPT trap from the User VM
+to
 DM, and the xHCI DM or DRD DM will emulate hardware behaviors to make
 the subsystem run.
 
```

Comment:
@NanlinXie The table below says it's current as of ACRN 1.4.  Can you get us an update that's good for v2.7?

Reply:
@NanlinXie who in engineering can help here?

Reply:
@NanlinXie said: [Nanlin] I suggest to remove this section since it should not exit in HLD.